OneOf.Serialization.Tests/OneOfJsonConverterTests.cs

```diff
@@ -21,6 +23,43 @@ public void Given_Instance_Of_OneOf_Case_When_Serializing_Then_Produces_Valid_Js
             var json = JsonConvert.SerializeObject(engine, settings);
             var engineJObject = JObject.FromObject(engine);
             json.Should().NotBeNullOrEmpty();
+            json.Should().StartWith("{").And.EndWith("}");
```

Comment:
@ra-mt what is the intention behind that? To verify we are dealing with an object?

Reply:
Yes, to verify that it's a JSON object.